src/components/Calculator/TotalScreen/TotalScreen.tsx

```diff
@@ -0,0 +1,24 @@
+import './totalScreen.css';
+
+import { type CalculatorState } from '../../../reducer';
+
+const INFINITY = 'Infinity';
+
+const TotalScreen = ({
+  calculatorState,
+}: {
+  calculatorState: CalculatorState;
+}) => {
+  const calculatorResult =
+    calculatorState.prevValue +
+    calculatorState.operator +
+    calculatorState.value;
```

Comment:
이 부분은 불필요하게 길어지는 느낌이 있네요! 객체를 한 번 더 분해할당하면 어떨까요?

```suggestion
const TotalScreen = ({
  calculatorState : { prevValue, operator, value },
}: {
  calculatorState: CalculatorState;
}) => {
  const calculatorResult = `${prevValue}${operator}${value}`;
```

문자열 덧셈보다는 템플릿 리터럴을 적극적으로 활용하는 게 읽기 편하답니다!